### PR TITLE
Init site template

### PR DESCRIPTION
**Pull request by shestera**

### CODE REVIEW

File: templates/_includes/footer.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+						<div class="contact__item">
+							<span class="contact__icon">
+								<svg class="icon icon-phone-footer">
+									<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#phone_footer"></use>
+								</svg>
+							</span>
+							<span class="contact__info">
+								{{ config.PHONE1 }} <br>
+								{{ config.PHONE2 }}6
```
Comment (sobolevn) [inline templates/_includes/footer.html]: Remove `6` at the end.

File: murkosha/settings/components/common.py [hunk 2]
```diff
@@ -237,6 +237,12 @@
 CONSTANCE_BACKEND = 'constance.backends.database.DatabaseBackend'
 
 CONSTANCE_CONFIG = {  # This is just an example:
-    'THE_ANSWER': (42, 'Answer to the Ultimate Question of Life, '
-                       'The Universe, and Everything'),
+    'ADDRESS1': ('Москва, м. Медведково,', 'ADDRESS1'),
```
Comment (sobolevn) [inline murkosha/settings/components/common.py]: Nice idea, but we use a different approach:
1. `contance` is used for hidden values. Like: `ADMIN_EMAIL`, `DEFAULT_EMAIL_SENDER` and so on
2. for static values it is possible to use `static_placeholder`

File: landing_app/templates/landing_app/landing.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{% extends '_layouts/base.html' %}
+{% load cms_tags sekizai_tags %}
+
+{% block content %}
+    {% placeholder "content" %}
```
Comment (sobolevn) [inline landing_app/templates/landing_app/landing.html]: Just one `placeholder` is fine just for the moment. Later it should be changed.

File: templates/_includes/top_header.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+					<span class="header-col__text">{{ config.INFO_EMAIL }}</span>
+					<span class="header-col__text">{{ config.PR_EMAIL }}</span>
+				</div>
+
+
+				<div class="header-col hidden-lg-down">
+					<svg class="header-col__icon icon-clock">
+						<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#clock"></use>
+					</svg>
+					<h5 class="title title-xs brand-color">{% trans "ВРЕМЯ РАБОТЫ" %}</h5>
```
Comment (sobolevn) [inline templates/_includes/top_header.html]: Things like that should be uppercased by `css`. Text values should have normal case.

**Comment by shestera** [comment]
Fixed remarks

**Pull request closed without merging**